#! /bin/sh -e

# DP: fastjar.texi: Directly define the gcctabopt and gccoptlist macros.

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        ;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

--- fastjar/fastjar.texi~	2006-02-08 05:55:47.000000000 +0000
+++ fastjar/fastjar.texi	2006-08-29 19:31:40.590718000 +0000
@@ -10,7 +10,16 @@
 @c When this manual is copyrighted.
 @set copyrights-fastjar 2002
 
-@include gcc-common.texi
+@macro gcctabopt{body}
+@code{\body\}
+@end macro
+@macro gccoptlist{body}
+@smallexample
+\body\
+@end smallexample
+@end macro
+
+@include gcc-vers.texi
 
 @c Versions
 @set which-gcj GCC-@value{version-GCC}
